testing/unitTestsBackendDockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ghcr.io/d-fine/dataland/temurinbase:latest
+
+WORKDIR /app
+
+CMD ["./gradlew","ktlintcheck","detekt",":dataland-backend:test"]
```

Comment:
Auch hier:
Auch hier gilt:
- Dieses File habe ich in Dala-50 gelöscht.
- Dala-50 wurde in Main gemergt
- Main wurde hier rein gemergt
- Wie kann es sein, dass das File wieder da ist?? Sehr ominös

Reply:
wurde gelöscht

Reply:
Ich glaub solche Fehler passieren wenn man bei lokalem Merge konflikten nicht vorsichtlich Sachen von einer Seite annimmt